MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,18 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        if ($(".validation-summary-errors").length > 0) {
+            scrollLogin(0)
+        } else {
+            setTimeout(scrollLogin, 1500, 900);
```

Comment:
could you please explain why you use two parameters here? 

And actually I can't understand why u do this here, could you please write some comment to explain that?